content/plugins/noxoua-activity-log.md

```diff
@@ -3,12 +3,12 @@ name: Activity Log
 slug: noxoua-activity-log
 author_slug: noxoua
 categories: [panel-builder]
-description: Add-on for simplified activity logging based on spatie/laravel-activitylog package.
+description: Add-on for simplified activity logging based on spatie package.
 discord_url: https://discord.com/channels/883083792112300104/1167950609970888774
-docs_url: https://raw.githubusercontent.com/noxoua/filament-activity-log/main/README.md
+docs_url: https://activity-log.noxo.app/
```

Comment:
This should be 'url' instead of 'docs_url' since it is not raw markdown.

Reply:
Okay, I would leave url to the readme file 👍